src/isDeepEqual.test.ts

```diff
@@ -155,6 +155,66 @@ describe("arrays", () => {
   });
 });
 
+describe("Maps", () => {
+  it("works on shallow equal maps", () => {
+    expect(isDeepEqual(new Map([["a", 1]]), new Map([["a", 1]]))).toBe(true);
+  });
+  test("two empty Maps should be equal", () => {
```

Comment:
why did you remove all the newlines?
```suggestion

  test("two empty Maps should be equal", () => {
```